### PR TITLE
Create Log Record Frontend Component

### PR DESCRIPTION
**Pull request by safchow**
## Notion ticket link

[Log Record Frontend Component](https://github.com/uwblueprint/supportive-housing/issues/29)

## Implementation description
* Created a new page, `LogRecords.tsx` that displays logs in a table.
* Created new endpoint, `/log-records` that will open up the page.
* Styled using lo-fi for font, font colour, font size, etc.

## Steps to test
1. Verify that going to the `/log-records` opens up the Log Records page. 
2. Verify that all information displays correctly, and that rows grow vertically if the text does not fit horizontally (refer to the fourth row for this).
3. Verify that styling matches the styling on the Figma lo-fis.

## What should reviewers focus on?
* Are there any styling changes that should be made?

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
- [x] If I have made API changes, I have updated the [REST API Docs](https://www.notion.so/uwblueprintexecs/REST-Endpoints-05ce60312bb943439dfda42bb1318536)
- [x] IF I have made changes to the db/models, I have updated the [Data Models Page](https://www.notion.so/uwblueprintexecs/Data-Models-760f8aa06b244eb0842c079ad77987b0)
- [x] I have updated other Docs as needed

### CODE REVIEW

File: frontend/src/App.tsx
Comment (DinuWije) [inline frontend/src/App.tsx:35]: Can you take a look at this warning? I'm not sure what it's trying to point out so if it's not important just ignore it. Also did you run the linters before pushing?

**Review by DinuWije — CHANGES_REQUESTED** [review]
Run linters and re-request a review pls. Other than that really great work, I'm super impressed! Once we get another review on this it should be good to go!!!!

File: frontend/src/App.tsx
Comment (safchow) [reply frontend/src/App.tsx:35]: I ran the front-end linter, but not the back-end linter. I'll run both and push

**Review by juthikahoque — APPROVED** [review]
this looks really great to me! i think it's good to merge once you guys deal w dinu's comments

**Review by DinuWije — APPROVED** [review]

**Pull request merged**
